ix-dev/stable/pihole/templates/docker-compose.yaml

```
@@ -6,18 +6,18 @@
 {% do c1.add_caps(["NET_ADMIN", "NET_RAW", "NET_BIND_SERVICE", "CHOWN", "DAC_OVERRIDE", "FOWNER", "SETGID", "SETUID", "SETFCAP", "SETPCAP", "KILL", "SYS_NICE"]) %}
 {% do c1.remove_security_opt("no-new-privileges") %}
 
-{% do c1.healthcheck.set_test("curl", {"port": values.network.web_port, "path": "/admin/login.php"}) %}
+{% do c1.healthcheck.set_test("curl", {"port": values.network.web_port, "path": "/admin/login"}) %}
 
-{% do c1.environment.add_env("WEB_PORT", values.network.web_port) %}
-{% do c1.environment.add_env("WEBPASSWORD", values.pihole.web_password) %}
+{% do c1.environment.add_env("FTLCONF_webserver_port", values.network.web_port) %}
+{% do c1.environment.add_env("FTLCONF_webserver_api_password", values.pihole.web_password) %}
 {% if values.network.get('dhcp_config', {}).get('dhcp_enabled', false) %}
-  {% do c1.environment.add_env("DHCP_ACTIVE", "true") %}
-  {% do c1.environment.add_env("DHCP_START", values.network.dhcp_config.start) %}
-  {% do c1.environment.add_env("DHCP_END", values.network.dhcp_config.end) %}
-  {% do c1.environment.add_env("DHCP_ROUTER", values.network.dhcp_config.gateway) %}
+  {% do c1.environment.add_env("FTLCONF_dhcp_active", "true") %}
+  {% do c1.environment.add_env("FTLCONF_dhcp_start", values.network.dhcp_config.start) %}
+  {% do c1.environment.add_env("FTLCONF_dhcp_end", values.network.dhcp_config.end) %}
+  {% do c1.environment.add_env("FTLCONF_dhcp_router", values.network.dhcp_config.gateway) %}
 {% endif %}
 {% if values.network.host_network and values.network.interface_name %}
-  {% do c1.environment.add_env("INTERFACE", values.network.interface_name) %}
+  {% do c1.environment.add_env("FTLCONF_dns_interface", values.network.interface_name) %}
```

If an interface is specified then `FTLCONF_dns_listeningMode` is set to single.
```suggestion
  {% do c1.environment.add_env("FTLCONF_dns_interface", values.network.interface_name) %}
  {% do c1.environment.add_env("FTLCONF_dns_listeningMode", "single") %}
```